### PR TITLE
Add a utility function to ensure an imagestream exists

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/assign @csrwng

### CODE REVIEW

File: vars/ensureImageStream.groovy [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/groovy
+
+def call(Object ctx, String name) {
+  echo "Ensuring image stream ${this._buildName} exists"
```
Comment (csrwng) [inline vars/ensureImageStream.groovy]: should use the name argument (same on line 9 below)

**Comment by csrwng** [comment]
one comment, otherwise lgtm

File: vars/ensureImageStream.groovy (hunk 1 above)
Comment (stevekuznetsov) [reply vars/ensureImageStream.groovy]: eek, sloppy

**Comment by stevekuznetsov** [comment]
Updated

**Pull request merged**